mllib/src/test/scala/org/apache/spark/ml/tree/impl/RandomForestSuite.scala

```diff
@@ -176,6 +203,18 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(tree.rootNode.impurity === -1.0)
```

Comment:
Hmm, my mistake. It seems that the only time that empty splits can occur are with regression and all continuous features. Unordered features can only occur for multiclass classification and for categorical features that have more than one category. So, there is no way to exercise the failure case with classification. I think we can just have the regression test.